core/src/main/resources/hudson/model/Job/configure.jelly

```diff
@@ -27,7 +27,7 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
-  <l:layout title="${it.displayName} Config" cssclass="main-panel-only" norefresh="true" permission="${it.EXTENDED_READ}">
+  <l:layout title="${it.displayName} Config" cssClass="main-panel-only" norefresh="true" permission="${it.EXTENDED_READ}">
```

Comment:
@jglick I've [checked](https://github.com/jenkinsci/jenkins/blob/2.0/core/src/main/resources/lib/layout/layout.jelly#L40) that the right name is `cssclass`. Reverted here https://github.com/jenkinsci/jenkins/pull/2117/commits/bc3846a9f93b0d73a03569e93a80e34e2ccaf573